css/main.css

```diff
@@ -233,33 +238,318 @@ textarea {
   a[href^="javascript:"]:after {
     content: "";
   }
+
   pre {
     white-space: pre-wrap !important;
   }
+
   pre,
   blockquote {
     border: 1px solid #999;
     page-break-inside: avoid;
   }
+
   /*
      * Printing Tables:
      * https://web.archive.org/web/20180815150934/http://css-discuss.incutio.com/wiki/Printing_Tables
      */
   thead {
     display: table-header-group;
   }
+
   tr,
   img {
     page-break-inside: avoid;
   }
+
   p,
   h2,
   h3 {
     orphans: 3;
     widows: 3;
   }
+
   h2,
   h3 {
     page-break-after: avoid;
   }
 }
+
+/*
+* Layout
+*/
+.content-container {
+  max-width: 960px;
+  margin: 0 auto;
+}
+
+.content-section {
+  margin-top: 5em;
+}
+
+/*
+* Utilities
+*/
+.text-center {
+  text-align: center;
+}
+
+/*
+* Typography
+*/
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+/*
+* Main Banner
+*/
+header {
+  height: 88px;
+}
+
+.logo {
+  width: 50px;
+  padding: 5px;
+}
+
+nav {
+  position: fixed;
+  z-index: 1;
+  background: linear-gradient(to right, rgba(204, 83, 51, 0.95), rgba(35, 7, 77, 0.95));
+  width: 100%;
+}
+
+nav .nav-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+ul {
+  width: 80%;
+  margin: 0 auto;
+  padding: 10px;
+}
+
+ul li {
+  list-style: none;
+  display: inline-block;
+  padding: 20px;
+  font-size: 20px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+ul li:hover {
+  background: cornflowerblue;
+}
+
+ul li a {
+  text-decoration: none;
+  color: white;
+}
+
+ul a:hover {
+  color: black;
+  border-bottom: 2px solid;
+  transition: color .5s, border-bottom .5s;
+}
+
+.toggle {
+  width: 100%;
+  padding: 10px 20px;
+  background: #3b8eec;
+  box-sizing: border-box;
+  text-align: right;
+  color: white;
+  font-size: 20px;
+  display: none;
+}
+
+@media (max-width: 768px) {
+  .toggle {
+    display: block;
+  }
+
+  ul {
+    width: 100%;
+    display: none;
+  }
+
+  ul li {
+    display: block;
+    text-align: center;
+  }
+
+  .active {
+    display: block;
+  }
+
+  .logo {
+    display: none;
+  }
+}
+
+/*
+* Image Carousel
+*/
```

Comment:
Each component should have the version number with it so we can keep track of updates.